### PR TITLE
website: adding more docs, details, blog post

### PR DESCRIPTION
**Pull request by asettle**
Signed-off-by: Alexandra Settle <asettle@suse.com>

### CODE REVIEW

**Comment by asettle** [comment]
Would like to see: https://github.com/aquarist-labs/aquarist-labs.github.io/issues/14 implemented before merging this. But not picky.

**Review by jecluis — APPROVED** [review]

File: _posts/2021-04-19-hello-world-blog.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+If you want to get your hands dirty as soon as possible, you can also run the
+script at `tools/setup-dev.sh`. This will ensure you have a basic development
+environment as soon as possible so you can start hacking.
+
+You will be able to find the backend bits in `src/gravel`, while the frontend
+bits are located in `src/glass`. We also have `wasser` (water) powering our CI.
+
+Check out our [From Zero to Hacking](https://github.com/aquarist-labs/aquarium/blob/main/doc/from-zero-to-hacking.md)
+quickstart to help you get off your feet.
```
Comment (tserong) [inline _posts/2021-04-19-hello-world-blog.md:25]: off your feet?

**Review by tserong — DISMISSED** [review]
LGTM with one tiny question

File: _posts/2021-04-19-hello-world-blog.md (hunk 1 above)
Comment (asettle) [reply _posts/2021-04-19-hello-world-blog.md:25]: LOL we need to change that in about 4 documents. GO ME.

**Review by jecluis — APPROVED** [review]

**Pull request merged**